packages/landing/src/attribution.ts

```diff
@@ -114,8 +118,17 @@ export class MapAttribution {
     // For example, 512×512 tiles at zoom level 4 are equivalent to 256×256 tiles at zoom level 5.
     this.zoom += 1;
 
+    const dateAfter = Config.map.dateRange.yearAfter
+      ? `${Config.map.dateRange.yearAfter}-01-01T00:00:00.000Z`
+      : undefined;
+    const dateBefore = Config.map.dateRange.yearAfter
+      ? `${Config.map.dateRange.yearBefore}-12-31T23:59:59.999Z`
+      : undefined;
     const bbox = this.mapboxBoundToBbox(this.bounds, Config.map.tileMatrix);
-    const filtered = attr.filter(bbox, this.zoom);
+    const filtered = attr.filter(bbox, this.zoom, dateAfter, dateBefore);
+    const filteredLayerIds = filtered.map((x) => x.id).join('_');
```

Comment:
Can we hash this instead of a very long string with ids.

Reply:
I did wonder about that, but wasn't sure how to create a hash without bringing in a dependency. What would be the cleanest way to do that?

Reply:
I normally use crypto in basemaps.

``import { createHash } from 'crypto'; 
const cacheKey = createHash('sha256').update(data).digest('base64');``

Reply:
this runs both in the browser and in node so `import crypto` is not a option, using a map join is a simple work around.